packages/clarity-js/src/layout/dom.ts

```diff
@@ -298,8 +299,8 @@ function updateSelector(value: NodeValue): void {
     let prefix = parent ? parent.selector : null;
     let d = value.data;
     let p = position(parent, value);
-    let s: SelectorInput = { tag: d.tag, prefix, position: p, attributes: d.attributes };
-    value.selector = [selector(s), selector(s, true)];
+    let s: SelectorInput = { id: value.id, tag: d.tag, prefix, position: p, attributes: d.attributes };
+    value.selector = [selector.get(s, Selector.Alpha), selector.get(s, Selector.Beta)];
     value.hash = value.selector.map(x => x ? hash(x) : null) as [string, string];
     value.hash.forEach(h => hashMap[h] = value.id);
     if (value.hash.some(h => extract.fragments.indexOf(h) !== -1)) {
```

Comment:
If I read this correctly it matches on either of the two types of hashes. Is that intentional? If so please just add a comment that is what this should do.

Reply:
Sounds good.